.gitignore

```diff
@@ -66,9 +66,16 @@ secret_key.txt
 # IDE / development files
 .idea/
 *.code-workspace
-.vscode/
 .bash_history
 
+# https://github.com/github/gitignore/blob/main/Global/VisualStudioCode.gitignore
+.vscode/*
+#!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
```

Comment:
Question: Won't this lead to a lot of changes in git? I have 50+ extensions in my full InvenTree dev enviroment

Reply:
I just used the template linked above, but we can comment this and uncomment as we need it. I guess in `extensions.json` we can specify extensions for usage without the devcontainer, as they are already specified in the `devcontainer.json`.